libbpfgo.go

```diff
@@ -776,6 +776,15 @@ func (b *BPFMap) GetValueReadInto(key unsafe.Pointer, value *[]byte) error {
 	return nil
 }
 
+func (b *BPFMap) SetInitialValue(value unsafe.Pointer) error {
```

Comment:
Is there a usecase for this outside of setting global variables? If not maybe we can do something like wrap the `initGlobalVariable()` function you have in the selftest into an API function.

Reply:
> Is there a usecase for this outside of setting global variables?

No, `bpf_map__set_initial_value` only support this usecase. 

> If not maybe we can do something like wrap the initGlobalVariable() function you have in the selftest into an API function.

I'll remove `SetInitialValue` and add a new API function: 

```go
func (m *Module) InitGlobalVariable(sectionName string, value unsafe.Pointer) error
```